ow_power_system/src/PowerSystemNode.cpp

```diff
@@ -284,7 +290,7 @@ void PowerSystemNode::initAndRun()
       = (
           actual_mech_power
           + m_electrical_power
-          + m_power_baseline_camera_controller
+          + m_power_baseline
           + electricalPowerLights()
         ) / (NUM_MODELS - m_deactivated_models);
```

Comment:
Outside the scope of this PR, but I find the double use of the term "model" quite confusing.  It refers to a GSAP instance in some places, and a battery node in others (with battery node meaning a bank of cells, 6 in this case).  @chetankul do you have thoughts on this?

Reply:
There's also the power system ROS node :)